### PR TITLE
映射文档 No. 46

### PR DESCRIPTION
**Pull request by xianyu11-x**
完成 https://github.com/PaddlePaddle/docs/issues/5691 中No.46的文档映射工作，其中新增8个api映射文档，有问题请联系

### CODE REVIEW

**Comment by xianyu11-x** [comment]
torch.diag和torch.meshgrid的API 映射已在仓库中存在
paddle.renorm的缺失对应的API文档，函数相关定义只在Release Note和对应的PR中存在

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.gcd.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## [仅 torch 参数更多]torch.gcd
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.gcd.md]: 分类名写为 torch参数更多 即可~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.gcd.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch | PaddlePaddle | 备注                                                |
+| ------- | ------------ | --------------------------------------------------- |
+| input   | x            | 表示输入的第一个 Tensor ，仅参数名不一致。          |
+| other   | y            | 表示输入的第二个 Tensor ，仅参数名不一致。           |
+| out     |              | 表示输出的 Tensor，Paddle 无此参数，需要进行转写。 |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.gcd.md]: 缺失的参数用 - 替代

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.histc.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+## [仅 torch 参数更多]torch.histc
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.histc.md]: 分类名写为 torch参数更多 即可

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.histc.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+torch.histc(input, bins=100, min=0, max=0, *, out=None)
+```
+
+### [paddle.histogram](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/histogram_cn.html#histogram)
+
+```python
+paddle.histogram(input, bins=100, min=0, max=0, name=None)
+```
+
+其中 Paddle 相比 PyTorch 支持更多其他参数，具体如下：
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.histc.md]: 应该是 `其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.histc.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+### 参数映射
+
+| PyTorch | PaddlePaddle | 备注                                                |
+| ------- | ------------ | --------------------------------------------------- |
+| input   | input        | 表示输入的 Tensor。                                  |
+| bins    | bins         | 表示直方图直条的个数。                              |
+| min     | min          | 表示范围的下边界。                                  |
+| max     | max          | 表示范围的上边界。                                  |
+| out     |              | 表示输出的 Tensor，Paddle 无此参数，需要进行转写。 |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.histc.md]: 缺失的参数用 - 替代

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.kron.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+### [paddle.kron](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/kron_cn.html#kron)
+
+```python
+paddle.kron(x, y, out=None, name=None)
+```
+
+两者功能一致，仅参数名不一致，具体如下：
+
+### 参数映射
+|PyTorch|PaddlePaddle|备注|
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.kron.md]: 每个单元格内容前后可以加几个空格，这样看着有些拥挤

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.lcm.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+## [仅 torch 参数更多]torch.lcm
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.lcm.md]: `torch 参数更多` 即可~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.lcm.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch | PaddlePaddle | 备注                                                |
+| ------- | ------------ | --------------------------------------------------- |
+| input   | x            | 表示输入的第一个 Tensor ，仅参数名不一致。          |
+| other   | y            | 表示输入的第二个 Tensor ，仅参数名不一致。           |
+| out     |              | 表示输出的 Tensor，Paddle 无此参数，需要进行转写。 |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.lcm.md]: 缺少的参数用 - 替代

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.logcumsumexp.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
+## [仅 torch 参数更多]torch.logcumsumexp
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.logcumsumexp.md]: torch 参数更多

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.logcumsumexp.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch | PaddlePaddle | 备注                                                                |
+| ------- | ------------ | ------------------------------------------------------------------- |
+| input   | x            | 表示输入的 Tensor ，仅参数名不一致。                                |
+| dim     | axis         | 表示需要计算的维度，仅参数名不一致。                                |
+| out     |              | 表示输出的 Tensor，Paddle 无此参数，需要进行转写。                 |
+|         | dtype        | 表示输出 Tensor 的数据类型，Pytorch 无此参数，Paddle 保持默认即可。 |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.logcumsumexp.md]: 缺少的参数用 - 替代

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.renorm.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
+## [仅 torch 参数更多]torch.renorm
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.renorm.md]: torch 参数更多

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.renorm.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch | PaddlePaddle | 备注                                                |
+| ------- | ------------ | --------------------------------------------------- |
+| input   | input        | 表示输入的 Tensor 。                                |
+| p       | p            | 表示 p-范数计算的 p 值。|
+| dim     | axis         | 表示切分的维度。                                    |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.renorm.md]: 备注可以补充一句 仅参数名不一致

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.renorm.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+| maxnorm | max_norm     | 表示子张量的 p-范数最大值，仅参数名不一致。          |
+| out     |              | 表示输出的 Tensor，Paddle 无此参数，需要进行转写。 |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.renorm.md]: 缺失的参数用 - 替代

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.repeat_interleave.md [hunk 14]
```diff
@@ -0,0 +1,23 @@
+## [仅 torch 参数更多]torch.repeat_interleave
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.repeat_interleave.md]: torch 参数更多

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.repeat_interleave.md [hunk 15]
```diff
@@ -0,0 +1,23 @@
...
+
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+|PyTorch|PaddlePaddle|备注|
+| ------- | ------- | ------- |
+| input   | x            | 表示输入的 Tensor ，仅参数名不一致。          |
+| repeats   | repeats    | 表示指定复制次数的 1-D Tensor 或指定的复制次数。           |
+| dim     |   axis        | 表示复制取值的维度，仅参数名不一致。 |
+| output_size     |          | 表示给定维度的总输出尺寸，Paddle 无此参数，一般对网络训练结果影响不大，可直接删除。 |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.repeat_interleave.md]: 缺失的参数可以用 - 代替

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.searchsorted.md [hunk 16]
```diff
@@ -0,0 +1,46 @@
+## [仅 torch 参数更多]torch.searchsorted
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.searchsorted.md]: torch 参数更多

**Review by Fancy-hjyp — COMMENTED** [review]
* 参数映射部分没有的参数可以用 - 代替
* 分类名可以直接写为 `torch 参数更多`，保持和其他文件的统一

**Comment by xianyu11-x** [comment]
所有问题均已修改更正。

**Review by Fancy-hjyp — APPROVED** [review]
LGTM

**Review by zhwesky2010 — APPROVED** [review]

**Comment by luotao1** [comment]
hi, @xianyu11-x  
- 非常感谢你对飞桨框架的贡献，我们正在运营一个PFCC组织，会通过定期分享技术知识与发布开发者主导任务的形式持续为飞桨框架做贡献，详情可见 https://github.com/luotao1 主页说明。
- 如果你对PFCC有兴趣，请发送邮件至 [ext_paddle_oss@baidu.com](mailto:ext_paddle_oss@baidu.com)，我们会邀请你加入~
